server/sonar-server/src/test/java/org/sonar/server/user/ws/CreateActionTest.java

```diff
@@ -121,6 +156,20 @@ public void reactivate_user() throws Exception {
       .setParam("password", "1234")
       .setParam("password_confirmation", "1234").execute()
       .assertJson(getClass(), "reactivate_user.json");
+
+    assertThat(service.getByLogin("john").login()).isEqualTo("john");
```

Comment:
Not really sure to understand this one. Wouldn't it be better to check the value of the "active" field ?